authorize_request_handler.go

```diff
@@ -158,6 +158,12 @@ func (f *Fosite) authorizeRequestParametersFromOpenIDConnectRequest(request *Aut
 	}
 
 	claims := token.Claims
+	// Reject the request if the "request_uri" authorization request
+	// parameter is provided.
+	if requestURI, _ := claims["request_uri"].(string); isPARRequest && requestURI != "" {
```

Comment:
I believe this can panic if the key does not exist

Reply:
This won't panic. The use of the `, _` ensures that. To confirm, I tried this with Go playground.

Reply:
You're right!